tracer/test/Datadog.Trace.Tests/Telemetry/ConfigurationTests.cs

```diff
@@ -39,6 +39,8 @@ public class ConfigurationTests
         "DD_GIT_COMMIT_COMMITTER_NAME",
         "DD_GIT_COMMIT_COMMITTER_EMAIL",
         "DD_GIT_COMMIT_COMMITTER_DATE",
+        "DD_ACTION_EXECUTION_ID",
+        "DD_PIPELINE_EXECUTION_ID",
```

Comment:
I'm a bit surprised you need these tbh 🤔 AFAICT you're not recording them in config... so not sure why they're necessary 🤔